### ISSUE TITLE
Can this work on the tty shell?

### ISSUE THREAD

**Issue by i30817** [eonpatapon/mpDris2#119]
Without ever entering 'gnome session  proper, or is that asking too much? I'm interested in using it in this mode and the fact it's using systemd gives me some hope that gnome is not hardwired in.

**Comment by grawity** [comment]
At its core, mpDris2 is a translator between the MPD protocol and the D-Bus based MPRIS protocol. There's nothing GNOME-related or even systemd-related in that.

However, if you want to use the additional features such as notifications or hotkeys, you'll need an environment that provides compatible notification or hotkey APIs – mpDris2 will not directly write notifications to a tty and it will not directly read key events from evdev.

Some or other libnotify (org.freedesktop.Notifications) provider is required for the former, and only the gnome-settings-daemon "media key" API is supported for the latter.

**Comment by i30817** [comment]
So i guess the question is 'can gnome-settings-daemon run outside of a gnome-shell session and not blow up the shell when started if yes'?

**Comment by grawity** [comment]
No, especially not in 3.18+ where it completely depends on gnome-shell.

Why do you need gnome-settings-daemon specifically? If you need hotkeys on a general X11 session, install xbindkeys/sxhkd and bind something to `mpc pause` and `mpc next`. If you need hotkeys on a bare tty console, no GNOME tools would help anyways.

**Comment by i30817** [comment]
I don't need it specifically. I just wanted the multimedia keys to be available outside of gnome, in the TTY, preferably with a easy path to creating a config where it's only active on TTYs, not after mode switch to the actual gnome with alt+ctrl+fkey.

Good hints about those, but i'm actually using wayland, not sure if they'd work. I guess i'm trying.
